.circleci/scripts/functorch_doc_push_script.sh

```diff
@@ -7,7 +7,7 @@ sudo apt-get -y install expect-dev
 
 # This is where the local pytorch install in the docker image is located
 pt_checkout="/var/lib/jenkins/workspace"
```

Comment:
Does this one need to stay as-is?

Reply:
Yea, since I haven't done anything for Jenkins -> CI which seems out of the scope here